src/components/UIShell/SideNav.js

```diff
@@ -46,16 +50,32 @@ export default class SideNav extends React.Component {
 
   static defaultProps = {
     translateById,
+    defaultExpanded: false,
   };
 
-  state = {
-    isExpanded: true,
-  };
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isExpanded: props.defaultExpanded,
```

Comment:
These states could be eventually conflicting, resulting in some weird bugs.

If the menu should always be expanded when it has focus, we don't want to create the opportunity for the state to fall out of sync. 

Perhaps we could stick to only using `isExpanded` and setting that based on the focus events.

Reply:
^ Makes sense, if we want to go with expand-on-focus. In this way, no additional style is needed.

Reply:
I was initially going to use the isExpanded state. But using that would require that we keep track of whether the user had it expanded or collapsed before focus so we could respect their preference after the item blurs, wouldn't it?

Reply:
As far as falling out of sync, I can imagine cases where the component can be in both states or just one so may be necessary for them to be out of sync for reason listed above.